books/free-programming-books-te.md

```diff
@@ -1,8 +1,14 @@
 ### Index
 
 * [C](#c)
+* [NPTEL](#nptel)
 
 
 ### <a id="c"></a>C
 
 * [Introduction to C \| Telugu](https://www.computerintelugu.com/2012/11/cmenu.html) - Sivanaadh Baazi Karampudi
+
+
+### NPTEL
+
+* [Books - Telugu](https://sites.google.com/nptel.iitm.ac.in/translated-ebook/telugu)
```

Comment:
Please add a book link directly instead of adding a repository. Remove this and add any of the books that are free and good